elide-core/src/main/java/com/yahoo/elide/annotation/DeletePermission.java

```diff
@@ -16,7 +18,7 @@
 /**
  * Assign custom Delete permission checks.
  */
-@Target({TYPE, PACKAGE})
+@Target({TYPE, PACKAGE, FIELD, METHOD})
```

Comment:
There is no way to Delete a field or method in JSON-API.  You can update relationships (by deleting elements) but that is an UpdatePermission.  I recommend we continue to use UpdatePermission for relationship management.

Reply:
Removed these changes. This was actually done so that the DeletePermission could be applied to the id field to filter off the operations by id on OpenAPI which I think was requested for HJSON as I think those endpoints to query by id don't work. I have changed it to use the Exclude annotation instead.